<launch>
  <arg name="veh"/>
  <arg name="package_namespace" default="apriltags2_ros" />
  <arg name="node_namespace" default="publish_detections_in_local_frame" />
  <arg name="topic_name" default="image_rect" />
  <!-- Set parameters -->
  <group ns="$(arg veh)/$(arg package_namespace)">
  <node pkg="apriltags2_ros" type="$(arg node_namespace).py" name="$(arg node_namespace)" clear_params="true" required="true" output="screen">
    <param name="veh" value="$(arg veh)"/>
  </node>
  </group>


</launch>
